Wrappers/Python/ccpi/optimisation/functions/MixedL21Norm.py

```diff
@@ -136,6 +95,8 @@ def proximal(self, x, tau, out=None):
             tmp = x.pnorm(2)
             res = (tmp - tau).maximum(0.0) * x/tmp
```

Comment:
for memory efficiency one may want to do
```python

res = (tmp -tau)
res.maximum(0.0, out=res)
res.divide(tmp, out=res)
res.multiply(x, out=res)
```
Looks terrible, though.

Reply:
Out will not work with maximum, divide etc, related to #523

Reply:
OK, let's fix that before, then

Reply:
#541 fixing #523 has been merged now. Merge master into this branch and you could use the `out` parameter in `BlockDataContainer` unary operations.